src/PopForums.Web/Views/Shared/_Layout.cshtml

```diff
@@ -1,12 +1,11 @@
 ﻿<!DOCTYPE html>
-<html>
+<html lang="@System.Threading.Thread.CurrentThread.CurrentUICulture.Name">
```

Comment:
This is really scope creep here... a PR shouldn't be reaching into all different kinds of places outside of the original problem domain.